.github/workflows/prettier.yml

```diff
@@ -0,0 +1,17 @@
+name: autofix.ci
+on:
+  pull_request:
+  push:
```

Comment:
I'm not sure if nested dictionary works intentionally or accidentally, but github's docs seem to imply a list is the expected shape: https://docs.github.com/en/actions/how-tos/write-workflows/choose-when-workflows-run/trigger-a-workflow#using-multiple-events

Reply:
Interesting. This is [directly from Prettier's docs](https://prettier.io/docs/ci), which could be out of date.

Reply:
okay I see both in the [Reference](https://docs.github.com/en/actions/reference/workflows-and-actions/workflow-syntax#using-activity-types-and-filters-with-multiple-events) so 🤷

Reply:
For instance, I think formatting should only be auto-applied in the trunk branch, I think otherwise the author should need to run prettier locally and author a commit.